FrontEnd/scripts/works.js

```diff
@@ -2,6 +2,7 @@
 let works = null;
 let categories = null;
 let idToken = sessionStorage.getItem("token");
+let uploadedImg = "";
```

Comment:
```suggestion
let uploadedImg = "";

```

let's add a bit of space ^^

Reply:
Is `uploadedImg` used?